compose/src/main/java/com/bottlerocketstudios/compose/snippets/snippetDetails/SnippetDetailsMockData.kt

```diff
@@ -0,0 +1,158 @@
+fun returnMockSnippetDetails() =
+    SnippetDetailsScreenState(
+        snippetDetails = SnippetDetailsUiModel(
+            id = "fake id",
+            title = "Test Snippet Title",
+            createdMessage = mockCreationMsg,
+            updatedMessage = mockUpdatedMsg,
+            isPrivate = false,
+            owner = User(
+                username = "Anakin",
+                nickname = "Annie",
+                accountStatus = "",
+                displayName = "Darth Vader",
+                createdOn = "",
+                uuid = "",
+                links = Links(null),
+                avatarUrl = "https://whatsondisneyplus.com/wp-content/uploads/2022/05/vader.png"
+            ),
+            creator = User(
+                username = "Obi-Wan",
+                nickname = "Ben",
+                accountStatus = "",
+                displayName = "Obi-Wan Kenobi",
+                createdOn = "",
+                uuid = "",
+                links = Links(null),
+                avatarUrl = "https://whatsondisneyplus.com/wp-content/uploads/2022/05/kenobi-avatar.png"
+            ),
+            httpsCloneLink = "https://bitbucket.org/snippets/jmichealmurray/zEqaLk/test-snippet-public.git",
+            sshCloneLink = "git@bitbucket.org:snippets/jmichealmurray/zEqaLk/test-snippet-public.git"
+        ).asMutableState(),
+        currentUser = currentUser.asMutableState(),
+        files = listOf(
+            SnippetDetailsFile(
+                fileName = "Test File Number 1",
+                links = Links(
+                    comments = Link("https://fake.snippet.com/comments", name = "")
+                )
+            ),
+            SnippetDetailsFile(
+                fileName = "Test File Number 2",
+                links = Links(
+                    comments = Link("https://fake.snippet.com/comments", name = "")
+                )
+            ),
+            SnippetDetailsFile(
+                fileName = "Test File Number 3",
+                links = Links(
+                    comments = Link("https://fake.snippet.com/comments", name = "")
+                )
+            ),
+            SnippetDetailsFile(
+                fileName = "Test File Number 4",
+                links = Links(
+                    comments = Link("https://fake.snippet.com/comments", name = "")
+                )
+            ),
+        ).asMutableState(),
+
+        isWatchingSnippet = false.asMutableState(),
+        onSnippetWatchClick = { },
+        onSnippetEditClick = { },
+        onSnippetDeleteClick = { },
+        comments = listOf(snippetComment).asMutableState(),
+        newSnippetComment = "".asMutableState(),
+        onCommentChanged =  {},
+        onSaveCommentClick = {},
+        onCancelNewCommentClick = {},
+        onDeleteCommentClick = {},
+        onEditCommentClick = {},
+        newReplyComment = "".asMutableState(),
+        onReplyChanged = {}
+    )
+
+fun ZonedDateTime.convertToTimeAgoMessage(): String {
```

Comment:
@MMurrayAtBR Since `compose` and `data` both declare this `ZonedDateTime.convertToTimeAgoMessage()` function (copy/pasted from one to the other), delete this code and move this to the `domain` module, which both `compose` and `data` modules depend on.